pkgs/by-name/ut/util-linux/package.nix

```diff
@@ -200,6 +200,24 @@ stdenv.mkDerivation rec {
     '';
 
   passthru = {
+    # Note for maintainers: if there is a hotfix we need to add and we want to prevent a mass rebuild,
+    # we can add it here, and reference it in the modules or packages that depend on it.
+    # util-linux transitively depends on util-linuxMinimal, and making a change in this file rebuilds
+    # the world. This lets us decouple hotfixes requiring mass rebuilds from fixing issues for one
+    # package or module that needs it.
+    #
+    # This was originally needed for Kubernetes (#405952 and #407810).
+    #
+    # Of course, keep this in sync with staging merges if we do end up needing anything.
+    withPatches = finalPackage.overrideAttrs (prev: {
+      patches = lib.unique (
```

Comment:
Note that this lib.unique is written in a way where, after the staging merge, withPatches and normal util-linux will eval to the same outpath.